flexget/plugins/input/rss.py

```diff
@@ -184,6 +185,25 @@ def process_invalid_content(self, task, data, url):
             f.write(data)
         log.critical('I have saved the invalid content to %s for you to view', filepath)
 
+    def escape_content(self, content):
+        valid_escapes = ('&quot;', '&apos;', '&lt;', '&gt;', '&amp;')
+        future_result = []
+        in_cdata_block = False
+
+        for idx, char in enumerate(content):
+
+            if type(char) is int:
+                char = chr(char)
+            if not in_cdata_block and char == '&':
+                if not str(content[idx:idx+7]).startswith(valid_escapes):
```

Comment:
This `str()` call should be removed. Once the literals you are comparing to are bytes, it won't be a problem.